model/testutil/api.go

```diff
@@ -75,19 +75,28 @@ func SetupAPITestData(testConfig *evergreen.Settings, taskDisplayName string, va
 
 	// Create the ref for the project
 	projectRef := &model.ProjectRef{
-		Identifier:  project.DisplayName,
-		Owner:       project.Owner,
-		Repo:        project.Repo,
-		RepoKind:    project.RepoKind,
-		Branch:      project.Branch,
-		Enabled:     project.Enabled,
-		BatchTime:   project.BatchTime,
-		LocalConfig: string(projectConfig),
+		Identifier: project.DisplayName,
+		Owner:      project.Owner,
+		Repo:       project.Repo,
+		RepoKind:   project.RepoKind,
+		Branch:     project.Branch,
+		Enabled:    project.Enabled,
+		BatchTime:  project.BatchTime,
 	}
 	if err = projectRef.Insert(); err != nil {
 		return nil, errors.Wrap(err, "failed to insert projectRef")
 	}
 
+	version := &model.Version{
+		Id:         "sample_version",
+		Identifier: project.DisplayName,
+		Requester:  evergreen.RepotrackerVersionRequester,
+		Config:     string(projectConfig),
```

Comment:
Why are we inserting this?

Reply:
This is purely for tests. Previously the `projectConfig` was set in `LocalConfig` of the `projectRef` (just above). Now, however, there is no `LocalConfig`, and the code looks for the last known good version. Inserting this version allows test code to find the config.